src/test/java/net/datafaker/idnumbers/SwedishIdNumberTest.java

```diff
@@ -14,6 +14,7 @@ void validSwedishSsn() {
 
     @Test
     void invalidSwedishSsn() {
+        assertThat(SwedenIdNumber.isValidSwedishSsn("020914-0001")).isFalse();
```

Comment:
I don't get this, you said 001 to 999 are fine for last 3. So why should 001 be isvalid false?

Or did you mean it should be 3 digits (excluding triple zero) following the hyphen not 4?

Reply:
I mean last 3 number without checksum digit so 020914-0001 is invalid.